assembl/static2/js/app/utils/server.js

```diff
@@ -0,0 +1,16 @@
+/** *
+ * Use the standard assembl server location for API calls or Stargate
+ *
+ * @return String path - the path to use for the API call
+ * */
+export const basePath = () => {
+  const useStargate = document.getElementById('useStargate') ? document.getElementById('useStargate').value : 'false';
```

Comment:
same comment as in `assembl/static2/js/app/client.js`: it's a bit confusing having a boolean as a string

Reply:
It's a field in HTML - I thought it was safer to pull as a string rather than boolean